Tests/ACKLocalizationCoreTests/ACKLocalization+Plurals.swift

```diff
@@ -0,0 +1,99 @@
+import XCTest
+@testable import ACKLocalizationCore
+
+final class ACKLocalizationPluralsTests: XCTestCase {
+    private var ackLocalization: ACKLocalization!
+    private var authAPI: AuthAPIServiceMock!
+    private var sheetsAPI: SheetsAPIServiceMock!
+    
+    // MARK: - Setup
+    
+    override func setUp() {
+        super.setUp()
+        
+        authAPI = AuthAPIServiceMock()
+        sheetsAPI = SheetsAPIServiceMock()
+        ackLocalization = ACKLocalization(authAPI: authAPI, sheetsAPI: sheetsAPI)
+    }
+    
+    // MARK: - Tests
+    
+    func testEmptyTranslations() {
+        let rows: [LocRow] = []
+        
+        let plurals = try! ackLocalization.buildPlurals(from: rows)
+        
+        XCTAssertEqual(plurals.count, 0)
+    }
+    
+    func testNoPlurals() {
+        let rows: [LocRow] = [
+            LocRow(key: "key1", value: "value1"),
+            LocRow(key: "key2", value: "value2"),
+            LocRow(key: "key3", value: "value3")
+        ]
+        
+        let plurals = try! ackLocalization.buildPlurals(from: rows)
+        
+        XCTAssertEqual(plurals.count, 0)
+    }
+    
+    func testOnePlural() {
+        let rows = [
+            LocRow(key: "key##{zero}", value: "zero"),
+            LocRow(key: "key##{one}", value: "one"),
+            LocRow(key: "key##{two}", value: "two"),
+            LocRow(key: "key##{many}", value: "many"),
+            LocRow(key: "key##{other}", value: "other")
+        ]
+        
+        let plurals = try! ackLocalization.buildPlurals(from: rows)
+        
+        XCTAssertEqual(plurals.count, 1)
+        XCTAssertEqual(Array(plurals.values)[0].translations.count, rows.count)
+    }
+    
+    func testMultiplePlurals() {
+        let rows = [
+            LocRow(key: "key##{zero}", value: "zero"),
+            LocRow(key: "key2##{one}", value: "one")
+        ]
+        
+        let plurals = try! ackLocalization.buildPlurals(from: rows)
+        
+        XCTAssertEqual(plurals.count, 2)
+        for plural in plurals.values {
+            XCTAssertEqual(plural.translations.count, 1)
+        }
+    }
+    
+    func testMissingTranslationKey() {
+        let rows = [
+            LocRow(key: "##{zero}", value: "zero")
+        ]
+        
+        XCTAssertThrowsError(try ackLocalization.buildPlurals(from: rows)) { error in
```

Comment:
might be nice to add a following extension for `XCTest` (might be handy in the future, too): 
```swift
    func XCTAssertThrowsSpecific<Error: Swift.Error & Equatable, T>(_ closure: @autoclosure () throws -> T, _ error: Error, file: StaticString = #file, line: UInt = #line) {
        do {
            _ = try closure()
        } catch let closureError as Error {
            XCTAssertEqual(error, closureError, file: file, line: line)
        } catch let closureError {
            XCTFail("\(error) is not equal to: \(closureError)", file: file, line: line)
        }
    }
```

Your code can then be changed to something like this:
`XCTAssertThrowsSpecific(try ackLocalization.buildPlurals(from: rows), PluralError.missingTranslationkey(rows[0].key))`